tests/phpunit/api/v3/CaseTypeTest.php

```diff
@@ -15,37 +15,37 @@
  */
 class api_v3_CaseTypeTest extends CiviCaseTestCase {
 
-  public function setUp(): void {
-    $this->quickCleanup(['civicrm_case_type']);
-    parent::setUp();
-
-    $this->fixtures['Application_with_Definition'] = [
-      'title' => 'Application with Definition',
-      'name' => 'Application_with_Definition',
-      'is_active' => 1,
-      'weight' => 4,
-      'definition' => [
-        'activityTypes' => [
-          ['name' => 'First act'],
-        ],
-        'activitySets' => [
-          [
-            'name' => 'set1',
-            'label' => 'Label 1',
-            'timeline' => 1,
-            'activityTypes' => [
-              ['name' => 'Open Case', 'status' => 'Completed'],
-            ],
+  const APPLICATION_WITH_DEFINITION_PARAMS = [
+    'title' => 'Application with Definition',
+    'name' => 'Application_with_Definition',
+    'is_active' => 1,
+    'weight' => 4,
+    'definition' => [
+      'activityTypes' => [
+        ['name' => 'First act'],
+      ],
+      'activitySets' => [
+        [
+          'name' => 'set1',
+          'label' => 'Label 1',
+          'timeline' => 1,
+          'activityTypes' => [
+            ['name' => 'Open Case', 'status' => 'Completed'],
           ],
         ],
-        'timelineActivityTypes' => [
-          ['name' => 'Open Case', 'status' => 'Completed'],
-        ],
-        'caseRoles' => [
-          ['name' => 'First role', 'creator' => 1, 'manager' => 1],
-        ],
       ],
-    ];
+      'timelineActivityTypes' => [
+        ['name' => 'Open Case', 'status' => 'Completed'],
+      ],
+      'caseRoles' => [
+        ['name' => 'First role', 'creator' => 1, 'manager' => 1],
+      ],
+    ],
+  ];
+
+  public function setUp(): void {
+    $this->quickCleanup(['civicrm_case_type']);
```

Comment:
It's unusual to put this here normally it goes it tearDown(), but whatever works.

Reply:
But may want to add a comment why it's needed in both.

Reply:
I didn't actually change any functionality here, just switched to a `const` - `$this->quickCleanup(['civicrm_case_type']);` was in `setUp` previously, so I'm not sure why it is why it is.

Reply:
oh i see